### ISSUE TITLE
Number of items no longer announced in tree views by NVDA

### ISSUE THREAD

**Issue by Neurrone** [microsoft/vscode#97359]
Issue Type: <b>Bug</b>

In explorer view, navigate through the tree view. 

Info about the number of items e.g, "node_modules 2 of 5" used to be read but somewhere in the past few months this is now gone.

This isn't that big of a deal but if there aren't technical reasons why this is not set I'd rather have it because this is actually useful in some cases. For example, working through a find and replace job across multiple files.

I have `explorer.compactFolders` set to `false` but that shouldn't affect this.

VS Code version: Code - Insiders 1.46.0-insider (d487078dc7fc1c276657cadb61b4f63833a8df55, 2020-05-07T16:19:54.327Z)
OS version: Windows_NT x64 10.0.18363

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz (8 x 1992)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off_ok<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.86GB (2.77GB free)|
|Process Argv||
|Screen Reader|yes|
|VM|40%|
</details><details><summary>Extensions (12)</summary>

Extension|Author (truncated)|Version
---|---|---
notes|can|1.0.0
vscode-markdownlint|Dav|0.35.1
vscode-eslint|dba|2.1.5
rest-client|hum|0.23.2
python|ms-|2020.4.76186
remote-wsl|ms-|0.44.2
cpptools|ms-|0.27.1
Go|ms-|0.14.1
vsnotes|pat|0.7.1
rust|rus|0.7.5
markdown-preview-enhanced|shd|0.5.3
vscode-webhint|web|1.5.5

</details>

**Comment by Neurrone** [comment]
CC: @isidorn

**Comment by isidorn** [comment]
@Neurrone thanks a lot for filling this. I can reproduce this with NVDA and Orca. We properly set the aria attributes `aria-setsize` and `aria-posinset`. So I am not sure why screen readers are not reading this content
@joanmarie can you maybe investigate on the Orca side?

It might be that our "weird" tree selection confuses screen readers, since we set aria-selected = false on focused elements.
fyi @leonardder @feerrenrut

**Comment by leonardder** [comment]
I think it would help to know where the regression started to occur. It could also be a bug in Chromium.

**Comment by Neurrone** [comment]
When focus was over one of the tree items, I did nvda+f1 to bring up debug info and this is what NVDA reports for the focused object:

```
```
roles:treeitem; level:4; setsize:0; posinset:0
```

This is probably the culprit -  a bug in how the values are computed.

**Comment by joanmarie** [comment]
Presumably this Chromium bug which is already fixed. https://bugs.chromium.org/p/chromium/issues/detail?id=1066632

In case someone reports something similar regarding the selected item count, that would presumably be this other Chromium bug which is already fixed.
https://bugs.chromium.org/p/chromium/issues/detail?id=1058961

**Comment by isidorn** [comment]
@joanmarie awesome!

My feeling is that this is not super important so we will not back port. Though please let me know if someone is passionate about this and we can see with @deepak1556 about potentially backporting the fix.

**Comment by deepak1556** [comment]
@joanmarie thanks for the links to the CL and also thanks for fixing them.

Have backported them in electron

https://github.com/electron/electron/pull/24061
https://github.com/electron/electron/pull/24060

**Comment by isidorn** [comment]
@deepak1556 thanks a lot for backporting. I see the backport has a 8 target, so we should have this once we update to electron 8?

**Comment by deepak1556** [comment]
yup I plan to push Electron 8 forward this iteration :) Should be in insiders this week.

**Comment by isidorn** [comment]
@deepak1556 awesome!

**Comment by isidorn** [comment]
As @deepak1556 informed me we plan to go to electron 9 in the next milestone, thus pushing this out to August.

**Comment by isidorn** [comment]
Seems like we will shipt with Electron 9, thus closing this optimistically.

**Comment by Neurrone** [comment]
This is still happening on latest insiders build (electron 9) on my other machine that does not have this yet.

Can this be reopened?

**Comment by Neurrone** [comment]
Oh wait sorry, my bad. I got confused between this issue and a separate one, which is not this one. Please ignore the previous comment.
